libkbfs/kbfs_ops_concur_test.go

```diff
@@ -1311,6 +1312,7 @@ func TestKBFSOpsCanceledCreateDelayTimeoutErrors(t *testing.T) {
 	// cancellation should have been enabled.
 	<-onPutStalledCh
 	cancel()
+	time.Sleep(20 * time.Millisecond) // make sure the cancellation is relayed
```

Comment:
Instead of a hard delay, what if you just did `<-ctx.Done()`, which will guarantee that the `DelayCancellation` object cancelled the context passed to `CreateFile`?